docs/monitors/host-metadata.md

```diff
@@ -37,12 +37,7 @@ Monitor Type: `host-metadata`
 
 ## Configuration
 
-| Config option | Required | Type | Description |
```

Comment:
This isn't backwards compatible.  The K8s configmap has these two values so there are going to be a lot of people using this config.

Reply:
ok, we'll leave them out and only use the top-level ones, esp since they can't be overridden on a per-monitor basis.